nwb_conversion_tools/datainterfaces/behavior/movie/moviedatainterface.py

```diff
@@ -84,6 +74,8 @@ def run_conversion(
         chunk_data: bool = True,
         module_name: Optional[str] = None,
         module_description: Optional[str] = None,
+        compression: Optional[str] = "gzip",
+        compression_options: Optional[dict] = None,
```

Comment:
```suggestion
        compression_options: Optional[int] = None,
```
As correctly indicated in the documentation, this only currently supports `int` values for `gzip`.